packages/platforms/browser/lib/auto-instrumentation/network-request-plugin.ts

```diff
@@ -48,6 +45,6 @@ export class NetworkRequestPlugin implements Plugin<BrowserConfiguration> {
   }
 
   private shouldTrackRequest (startContext: RequestStartContext): boolean {
-    return !this.ignoredUrls.some(url => url.test(startContext.url))
+    return !startContext.url.includes(this.configEndpoint)
```

Comment:
Does this need to be an `includes`? We make requests to the exact endpoint so it feels like `===` would be fine?

Reply:
that's a good point - I think I was assuming the possibility of additional paths or query strings. strict equality would be fine here.